### PR TITLE
Adding tags to Enforce docs

### PR DESCRIPTION
**Pull request by sheesh**
## Type of change
Added tags to Enforce articles.  

### What should this PR do?
Tags should appear in Enforce articles. 

### Why are we making this change?
Implementing tags is part of the Academy V2 design plans. 

### What are the acceptance criteria? 
Tags should appear on relevant content at the top of the article. Nothing should break. 

### How should this PR be tested?
The reviewer should preview a few Enforce tutorials to make sure tags are appearing correctly.

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ description: "Writing Rego-based policies for Chainguard Enforce"
 date: 2023-01-12T15:56:52-07:00
 lastmod: 2023-01-12T15:56:52-07:00
 draft: false
-tags: ["SBOM"]
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md:8]: The ones I tagged with SBOM do have SBOM policies. I don't know how granular you would like to be

**Review by ltagliaferri — APPROVED** [review]
Just a note about how granular you want tags

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md (hunk 1 above)
Comment (sheesh) [reply content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/chainguard-enforce-rego-policies.md:8]: Ah, I was thinking tagging at a higher level. But it makes sense to tag these with SBOMs. I'll add them back in,

**Comment by sheesh** [comment]
Hey @jamonation - this PR on Enforce tags now has a bunch of conflicts. Wondering if it's relevant to the autodocs work you were doing? Let me know if you have thoughts about how to resolve them. I don't want to mess anything up! :)  https://github.com/chainguard-dev/edu/pull/485/conflicts

**Pull request merged**